packages/frontend/pages/_app.tsx

```diff
@@ -75,6 +76,8 @@ function MyApp({ Component, pageProps }: any) {
 const Init = () => {
   useUpdateSqueethPrices()
   useUpdateSqueethPoolData()
+  useIndex()
```

Comment:
Instead of importing each function 1 by 1, how about an init function in the controller hook, export that, and use it here?

```
export const useInitController = () => {
  useIndex()
  useMark()
  useCurrentImpliedFunding()
  useDailyHistoricalFunding()
  useNormFactor()
}
```

Reply:
Also we can remove export from function like useIndex(), useMark() so no one will accidentally call that instead of using atoms